### ISSUE TITLE
exposure documentation for infrastructure

### ISSUE THREAD

**Issue by stufraser1** [GFDRR/rdl-docs#2]
exposure documentation (examples, taxonomy) relates only to buildings - we should include examples and codes possible to use for infrastructure too

**Comment by pzwsk** [comment]
Agree, and the page https://docs.riskdatalibrary.org/exposure.html should be added to the Exposure section.

**Comment by stufraser1** [comment]
The attributes given in the table and examples are only part of what can be completed. I think the value fields should also be included in the table explicitly, rather than mentioned in the text only.
